docs/configuration.md

```diff
@@ -556,6 +576,28 @@ value: <tmpl_string>
 [ short: <boolean> | default = slack_config.short_fields ]
 ```
 
+## `<matcher>`
+
+A matcher is a string with a syntax inspired by PromQL and Openmetrics. The syntax of a matcher consists of three tokens: 
+- A valid Prometheus label name. 
+- One of '=', '!=', '=~', or '!~', with the same meaning as known from PromQL selectors. 
```

Comment:
Perhaps use backticks rather than single quotes to render it nicely in Markdown?

Reply:
And perhaps give a short explanation what the operators mean. Not every person writing Alertmanager configs knows PromQL.

Reply:
Done